frontend/.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 pack
+coverage
+jest.results.json
```

Comment:
This was just for an action we aren't using, so can be gotten rid of. I'd like to keep coverage here though if we ever run tests purely locally.